spec.html

```diff
@@ -47268,7 +47268,7 @@ <h1>FinalizationRegistry.prototype</h1>
 
     <emu-clause id="sec-properties-of-the-finalization-registry-prototype-object">
       <h1>Properties of the FinalizationRegistry Prototype Object</h1>
-      <p>The <dfn>FinalizationRegistry prototype</dfn> object:</p>
+      <p>The <dfn>FinalizationRegistry prototype object</dfn>:</p>
```

Comment:
This un-defines the term "FinalizationRegistry prototype" (sans "object"), which is used in https://tc39.es/ecma262/#sec-properties-of-finalization-registry-instances. We either need to update that reference or add a `variants` to this `dfn`.

Reply:
a variant would be better, to preserve the old link

Reply:
Since the generated link links to the containing section instead of an anchor for this term (it would link to the term if it had an `id` of its own), there's no ID that would need an oldid here.